libcodechecker/cmd/cmd_line_client.py

```diff
@@ -321,7 +327,79 @@ def get_diff_report_dir(client, baseids, report_dir, diff_type):
                     filtered_reports.append(result)
         return filtered_reports
 
-    def print_reports(client, reports, output_format):
+    def cache_report_file(file_cache, file_id):
```

Comment:
I do not like the name of this function. The main purpose is not to cache the file but to lookup a file based on a file id.

I would rename it to something like `cached_report_file_lookup` or `get_report_file`.